panel/dist/css/divider.css

```diff
@@ -0,0 +1,10 @@
+:host{
+    width: 100%;
+} 
+div.bk-clearfix{
+    width:100%;
+}
+hr{
+    margin:0px;
+    margin-bottom:3px;   
```

Comment:
The margin-bottom is to center the `hr` element vertically. Between the panel above and below.